### PR TITLE
Add labels to sf station config

### PR DESCRIPTION
**Pull request by egid-fertal-inno**
Add configuration for watchlist Member labels

### CODE REVIEW

File: Chart.yaml [hunk 1]
```diff
@@ -36,6 +36,6 @@ maintainers:
 - name: Innovatrics
   url: https://www.innovatrics.com/
 
-version: 0.8.3
+version: 0.8.4
```
Comment (FooRider) [inline Chart.yaml:39]:
```suggestion
version: 0.8.4-alpha1
```

File: values.yaml [hunk 2]
```diff
@@ -66,6 +66,11 @@ configurations:
     # -- config containing authorization configuration for SF Station used when authentication is enabled for SF Station
     configName: "station-auth-config"
     secretName: "station-client-id"
+  watchlistMemberLabels:
```
Comment (FooRider) [inline values.yaml:69]:
```suggestion
  watchlistMemberLabels: []
```

File: README.md [hunk 3]
```diff
@@ -282,6 +282,7 @@ metadata:
 | configurations.s3.useBucketRegion | bool | `true` | mechanism to resolve bucket endpoint - if `true` then connection is made based on bucket region. If `false` then bucket endpoint needs to be set manually |
 | configurations.stationAuth.configName | string | `"station-auth-config"` | config containing authorization configuration for SF Station used when authentication is enabled for SF Station |
 | configurations.stationAuth.secretName | string | `"station-client-id"` |  |
+| configurations.watchlistMemberLabels | string | `nil` |  |
```
Comment (FooRider) [inline README.md:285]: Add description "Array of pre-defined watchlist member labels"

**Review by DanielRosival — APPROVED** [review]

File: values.yaml (hunk 2 above)
Comment (egid-fertal-inno) [reply values.yaml]: done

File: README.md (hunk 3 above)
Comment (egid-fertal-inno) [reply README.md]: done

**Review by FooRider — APPROVED** [review]

**Pull request merged**